docs/current-cli-contract.md

```
@@ -357,38 +384,82 @@ unchanged.
   while preserving the same effective
-  configuration facts. The `previous offsets` row reports only the effective
-  config mode: `disabled`, `prompt`, or `always`.
-- The `analysis mode` row reports the effective `analysis.performance_mode`:
-  `quality` or `performance`.
-- The at-a-glance workspace paths show `root`, `config`, `input`, and the resolved
+  configuration facts. The `previous offsets` row reports status-prefixed
+  explanatory text: `do not reuse previous offsets (disabled)`, `ask before
+  reusing previous offsets (prompt)`, or `reuse previous offsets when valid
+  (always)`.
+- The `analysis mode` row reports the effective `analysis.performance_mode` as
+  `quality` or `performance` and appends ` (skipped for this run)` when
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)